app/src/lib/git/update-ref.ts

```diff
@@ -36,17 +36,13 @@ export async function updateRef(
 export async function deleteRef(
   repository: Repository,
   ref: string,
-  reason: string
-): Promise<true | undefined> {
-  const result = await git(
-    ['update-ref', '-m', reason, '-d', ref],
-    repository.path,
-    'deleteRef'
-  )
+  reason?: string
```

Comment:
What's the reasoning for making this argument optional? i don't think its critical, but seems like a nice bit of logging, just in case. (non-blocking)

Reply:
Adding a reflog entry when deleting a ref doesn't do much in terms of logging because the reflog file for that ref will get deleted as well.

The reason we have the `reason` parameter at all is because we previously only used it to delete the `HEAD` reference when undoing the initial commit which is a super special case because the HEAD reflog file will never get deleted so we're able to tack on a message as part of that operation.

I'll update the parameter documentation to make this clear

Reply:
gotcha, that makes sense! thank you!